vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors_moe.py

```diff
@@ -70,6 +74,8 @@ def get_moe_method(
               and layer.activation == "silu" and layer.expert_map is None):
             return CompressedTensorsW8A8Fp8MoECutlassMethod(quant_config)
         elif quant_config._is_fp8_w8a8(weight_quant, input_quant):
+            if is_rocm_aiter_channel_scaled_moe_enabled():
```

Comment:
`tkw1` is not a general support of FP8 FMOE channel / rowwise scaling, it only supports the case when `apply_router_weight_on_input =True`